www.foia.gov/reports.html

```diff
@@ -58,14 +56,14 @@
 
   </div>-->
   <div id="reportsGlance" >
-  <h2>Most Popular</h2><ol><div class="reportText">The most popular reports run by users of FOIA.gov</div><li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DHS&amp;requestYear=2016">Department of Homeland Security: Requests 2016</a></li>
-<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DOJ&amp;requestYear=2016">Department of Justice: Requests 2016</a></li>
-<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DoD&amp;requestYear=2016">Department of Defense: Requests 2016</a></li>
-<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DHS&amp;requestYear=2015">Department of Homeland Security: Requests 2015</a></li>
-<li><a href="data.html?foia/Services/RequestDisposition.jsp?agencyName=DOJ&amp;requestYear=2016">Department of Justice: Requests Disposition 2016</a></li>
-<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DHS&amp;requestYear=2012">Department of Homeland Security: Requests 2012</a></li>
-<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DHS&amp;requestYear=2014">Department of Homeland Security: Requests 2014</a></li>
-<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DHS&amp;requestYear=2009">Department of Homeland Security: Requests 2009</a></li>
+  <h2>Most Popular</h2><ol><div class="reportText">The most popular reports run by users of FOIA.gov</div><li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DHS&amp;requestYear=2016">Department of Homeland Security: Requests 2016</a></li>
+<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DOJ&amp;requestYear=2016">Department of Justice: Requests 2016</a></li>
+<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DoD&amp;requestYear=2016">Department of Defense: Requests 2016</a></li>
+<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DHS&amp;requestYear=2015">Department of Homeland Security: Requests 2015</a></li>
+<li><a href="data.html?foia/Services/RequestDisposition.jsp?agencyName=DOJ&amp;requestYear=2016">Department of Justice: Requests Disposition 2016</a></li>
+<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DHS&amp;requestYear=2012">Department of Homeland Security: Requests 2012</a></li>
+<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DHS&amp;requestYear=2014">Department of Homeland Security: Requests 2014</a></li>
+<li><a href="data.html?foia/Services/DataRequest.jsp?agencyName=DHS&amp;requestYear=2009">Department of Homeland Security: Requests 2009</a></li>
```

Comment:
For the life of me I can't figure out how these changed.

Reply:
It's probably the line endings dos -> unix.